src/retrace-server-task

```diff
@@ -17,6 +17,7 @@ from typing import Any, Dict, Optional, Tuple
 import requests
 from requests.packages.urllib3.exceptions import InsecureRequestWarning
 from requests_gssapi import HTTPSPNEGOAuth, DISABLED
+from requests.exceptions import SSLError
```

Comment:
This import is unnecessary since you refer to the exception by the fully qualified name down below.

Reply:
Removed